src/getVar.js

```diff
@@ -43,8 +41,7 @@ var specialPropertyAccessors = {
  * @param supportSpecial
  * @returns {*}
  */
-var getObjectProperty = function(host, path, supportSpecial) {
-  var propChain = path.split('.');
+var getObjectProperty = function(host, propChain, supportSpecial) {
   var value = host;
   var attrMatch;
   for (var i = 0, len = propChain.length; i < len; i++) {
```

Comment:
Do you have to support IE older than version 9?  Using Array.forEach() make for better readability and less variable assignlengths/cleaner code.  changing this to a forEach you get rid of `var i = 0, len = propChain.length`